rasa/nlu/featurizers/dense_featurizer/convert_featurizer.py

```diff
@@ -98,6 +120,27 @@ def __init__(self, component_config: Optional[Dict[Text, Any]] = None) -> None:
             "default", self.module
         )
 
+    @classmethod
+    def validate_config(cls, config: Dict[Text, Any]) -> None:
```

Comment:
that's already checked as part of https://github.com/RasaHQ/rasa/blob/8d7a9cbfb058636cfaf22c6d495f16ddb9e12eb9/rasa/engine/validation.py#L134